docs/excel/custom-functions-best-practices.md

```diff
@@ -71,6 +71,8 @@ Keep in mind the following best practices when creating custom functions in your
 
 * In the JSON metadata file, specify the value of each `id` property in uppercase. Doing so makes it obvious which part of the `CustomFunctionMappings` statement in your JavaScript code corresponds to the `id` property in the JSON metadata file (provided that your function name uses camelCase, as recommended earlier).
 
+* It is required that the value of each `id` property contain alphanumeric characters. `id` property values *may* also contain periods. 
```

Comment:
suggest rephrasing to:

In the JSON metadata file, ensure that the value of each `id` property contains only alphanumeric characters and periods.